entity-management-mongo/src/main/resources/fulltext.properties

```diff
@@ -0,0 +1,41 @@
+# Fulltext properties file
+# Any sensitive data such as username/password settings must be overriden in a file titled fulltext.user.properties
+
+spring.application.name=Fulltext API
```

Comment:
this file belongs to the fulltext api, please remove

Reply:
Removed.